### ISSUE TITLE
User story - Expected UI/UX

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#46]
### User Story

As a **user**, I can **find pages such as a homepage, contact page, or see a 404 page when the page cannot be found** so that I can **have a familiar user experience**.

### Acceptance Criteria

- All links to pages should lead to the page or 404
- There should be a landing page, contact page, careers page, about us page, 500 error page and 404 page

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [x] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Create homepage
- [x] Create contact page
- [x] Create About us page
- [x] Create careers page
- [x] Create 404 page
- [x] Create 500 page
